db_master_deploy/README.md

```diff
@@ -0,0 +1,80 @@
+database managment scripts
+==========================
+These scripts have been developped for the database and table duplication on the postgres master instance.
+The scripts have to be executed with ``geodata`` user. To become geodata you have to type:
+```bash
+$ sudo su - geodata
+```
+##Installation
+Clone the git repository to geodata's home:
+```bash
+$ git clone git@github.com:geoadmin/deploy.git
+$ cd deploy/db_master_deploy/
+```
+Make sure that geodata .pgpass file exists. It should contain the credentials for superuser ``pgkogis``
+```bash
+$ echo "*:5432:*:pgkogis:xxxxxxxx" >> /home/geodata/.pgpass
+$ chmod 0600 /home/geodata/.pgpass
+```
+
+The script output will be written to syslog ``/var/log/messages`` and can be analyzed with kibana interface.
+
+###database naming convention and service address
+The read-only hot-standby slaves can be accessed via this hostname: ``pg.bgdi.ch``
+Each database has to have one of the following suffixes
+
+DB Suffix    | Meaning
+-------------|------------|
+**_master** |  Original database, will be continuosly updated, should not be used in any applications |
+**_dev**   | development copy of _master database  | 
+**_int**   | integration copy of _master database | 
+**_prod**  | production copy of _master database  | 
+**_demo** |  demo copy of _master database |
+**_YYYYMMDD** | timestamped copy of _master database (bod only, use parameter -a YYYYMMDD for the archive suffix) |
+
+###copy databases and or tables (deploy.sh)
+You can copy a comma delimited list of tables and/or database to one target. 
+The following targets can be used ``dev int prod demo``. 
+Normally we are deploying from a _master datasource to one of these targets. If you choose another source, the script will ask you for confirmation.
+Table copy is performance optimized sql copy from stdout to stdin. 
+The job will be using all the available cores by splitting up the table into  parts with equal number of rows, indexes and constraints will be removed before writing the table and re-created afterwards.
+
+####Examples for table copies
+```bash
+$ bash deploy.sh -s stopo_master.tlm.strasse -t int
+```
+```bash
+$ bash deploy.sh -s stopo_master.public.grid_ta25,bafu_master.prtr.swissprtr -t dev
+```
+The script fires the sphinx trigger script dml_trigger.sh
+
+Database copy is is an optimized version of ``createdb -T source_db``.
+
+####Examples for database copies
+Deploy bod to integration:
+```bash
+$ bash deploy.sh -s bod_master -t int
+```
+Deploy bod to production and make an archive copy, the script will create a new database named: ``bod_master20150315``
+```
+$ bash deploy.sh -s bod_master -t prod -a 20150315
```

Comment:
good point.
i will add a pure snapshot/archive option for the bod:
```
bash deploy.sh -s bod_master -a 20150315
```
will just create an archive of the bod ``bod_master20150315``.